script/deps.py

```diff
@@ -65,23 +69,49 @@ def EnsureDirExists(path):
         os.makedirs(path)
 
 
+def ReadETag(output_dir):
```

Comment:
really nice

Reply:
had to remove this from this PR, but we can add etag support for deps as a separate thing.